docs/deployment/create-account-and-orgs.md

```diff
@@ -0,0 +1,175 @@
+---
+slug: create-account-and-orgs
+append_help_link: true
+title: Create an account
+hide_title: true
+description: Create a Semgrep account and organization to prepare your deployment for the addition of repositories and team members.
+tags:
+  - Semgrep Cloud Platform
+  - Team & Enterprise Tier
+  - Deployment
+---
+
+import PlatformSigninGithub from "/src/components/procedure/_platform-signin-github.md"
+import PlatformSigninGitlab from "/src/components/procedure/_platform-signin-gitlab.md"
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Create a Semgrep account and set up organizations
+
+:::note Your deployment journey
+- You have gained the necessary [resource access and permissions](/deployment/checklist) required for deployment.
+:::
+
+* Add the rest of your organization (org) members to Semgrep. 
+* Configure Semgrep to scan repositories in other source code managers, such as Bitbucket.
+
+:::tip Using SSO for your initial sign-in 
+Alternatively, reach out to [<i class="fa-regular fa-envelope"></i> sales@semgrep.com](mailto:sales@semgrep.com) to set up SSO. This removes the need to sign in through a GitHub or GitLab account if you don't have one.
+:::
+
+## Semgrep Cloud Platform
+
+Semgrep Cloud Platform is used to manage all Semgrep Pro products. With SCP, you can:
+
+- View and manage your Semgrep findings.
+- Customize how Semgrep scans your code.
+- Manage the users associated with your Semgrep organization.
+- Set up alerts and notifications, including Slack alerts, emails, and pull request or merge request comments pushed to your source code manager
+
+## Initial sign in to Semgrep Cloud Platform
+
+The following steps walk you through creating a **user account** and your first **organization**:
+
+<Tabs
+    defaultValue="signin-github"
+    values={[
+    {label: 'Sign in with GitHub', value: 'signin-github'},
+    {label: 'Sign in with GitLab', value: 'signin-gitlab'},
+    ]}
+>
+
+<TabItem value='signin-github'>
+
+<PlatformSigninGithub />
+
+</TabItem>
+
+<TabItem value='signin-gitlab'>
+
+<PlatformSigninGitlab />
+
+</TabItem>
+
+</Tabs>
+
+You have successfully created an account and your first organization.
+
+## Set up organizations
+
+Organizations (orgs) in Semgrep enable users to share access to, and management of, Semgrep resources such as findings and reports.
+
+Semgrep organizations can be **connected** to equivalent GitHub, GitLab, and SSO organizations, which enables users from those organizations to easily join your Semgrep deployment through their existing credentials.
+
+### Next steps for GitHub and GitLab users
+
+- Connect your Semgrep org to your GitHub or GitLab SCM. Refer to [<i class="fa-regular fa-file-lines"></i> Connect a source code manager](/deployment/connect-an-org) for steps.
+
+### Next steps for Bitbucket and Azure Repos users
```

Comment:
Do we wan to specifically name BitBucket and Azure or just put them under "other SCM" users?

Reply:
I mention BB and Azure because they're confirmed to work with Semgrep; BB has partial support for core Semgrep features (i.e. the PR comments) and users will probably search for those terms. If we are aware of any other SCMs that have been confirmed to work please let me know?